Validation/RecoTrack/python/MultiTrackValidator_cfi.py

```diff
@@ -5,7 +5,8 @@
 from SimTracker.TrackAssociation.CosmicParametersDefinerForTP_cfi import *
 from Validation.RecoTrack.MTVHistoProducerAlgoForTrackerBlock_cfi import *
 
-multiTrackValidator = cms.EDAnalyzer(
+from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
+multiTrackValidator = DQMEDAnalyzer(
```

Comment:
Do I understand correctly that this PR does not change the behaviour of `DQMGlobalEDAnalyzer` in any way (i.e. those still need to be declared as `cms.EDAnalyzer`)? In that case, could you please leave MTV (also the "GenPs" variant above) out from this migration as #21794 already converts it to `DQMGlobalEDAnalyzer`?

Reply:
Yes, `DQMGlobalEDAnalyzer` is not affected.
No, if this code does run in "normal" relval workflows, and is not converted yet, it would crash the tests.

So we have an unfortunate conflict here. #21871 was supposed to buffer that a bit, but as you see I missed some declarations there... In the worst case, I can factor out the config changes here and make another "noop" PR like #21871.

Reply:
Nah, let's treat this as a normal conflict then, i.e. whichever of the two gets merged last reacts.